build.sbt

```diff
@@ -20,21 +20,14 @@
 name := "s2graph"
 
 lazy val commonSettings = Seq(
-  organization := "com.kakao.s2graph",
+  organization := "org.apache.s2graph",
   scalaVersion := "2.11.7",
-  version := "0.12.1-SNAPSHOT",
   scalacOptions := Seq("-language:postfixOps", "-unchecked", "-deprecation", "-feature", "-Xlint"),
   javaOptions ++= collection.JavaConversions.propertiesAsScalaMap(System.getProperties).map { case (key, value) => "-D" + key + "=" + value }.toSeq,
   testOptions in Test += Tests.Argument("-oDF"),
   parallelExecution in Test := false,
   resolvers ++= Seq(
-    Resolver.mavenLocal,
-    "Typesafe Repository" at "http://repo.typesafe.com/typesafe/releases/",
-    "Cloudera" at "https://repository.cloudera.com/artifactory/cloudera-repos",
-    "Twitter Maven" at "http://maven.twttr.com",
-    "sonatype-snapshots" at "https://oss.sonatype.org/content/repositories/snapshots",
-    "Scalaz Bintray Repo" at "http://dl.bintray.com/scalaz/releases",
-    "sonatype-releases" at "https://oss.sonatype.org/content/repositories/releases/"
+    Resolver.mavenLocal
```

Comment:
I tested with `sbt package` to see if we can package without those repos, and yes it can be packaged. I think all artifacts s2graph use can be resolved without those repos.